app/src/main/java/com/fpvout/digiview/MainActivity.java

```diff
@@ -113,6 +117,20 @@ private void connect(){
         mUsbMaskConnection.setUsbDevice(usbManager.openDevice(usbDevice), usbDevice);
         mVideoReader.setUsbMaskConnection(mUsbMaskConnection);
         mVideoReader.start();
+        mVideoReader.mPlayer.addListener(new ExoPlayer.EventListener() {
```

Comment:
can you merge this with the already declared event listener in VideoReaderExoplayer class (in start function / line 64) please ?
Better keep event handling on the player at the same place :)

Reply:
Done!